.gitignore

```diff
@@ -4,3 +4,5 @@
 *.orig
 .vscode
 */urdf/out*
+_build
```

Comment:
is `_build` a common folder for the docs build files?

Reply:
I don't have a lot of experience with building documentation for python packages, but "_build" seems to be the default name people use for sphinx-generated documentation.